autode/bracket/dhs.py

```diff
@@ -394,18 +395,25 @@ def ts_guess(self) -> Optional["Species"]:
         # jumping over the barrier. So we iterate through all coords
 
         energies = []
+        max_en = PotentialEnergy(-np.inf)
```

Comment:
`e` or `energy` would be more consistent

Reply:
changed it